### PR TITLE
#56 - Minor design updates

### PR DESCRIPTION
**Pull request by Koushith**
## Description

 - [x] Fixed input overflow issues.
 - [x] Replaced the text in shimmers.
 - [x] Added Favicon icons in manifest.json file.
 - [x] Navbar responsiveness 
## Screenshot
image
image
image

### CODE REVIEW

File: src/screens/wallet-overview/components/wallet-overview-shimmer.component.tsx [hunk 1]
```diff
@@ -43,7 +48,7 @@ export const WalletOverviewShimmer = () => {
             {Actions.map((items) => (
               <Box hCenter vCenter gap={1} key={items.id}>
                 <Square />
-                <Text variant='small' text={items.title} />
+                <SmallLine />
```
Comment (koshikraj) [inline src/screens/wallet-overview/components/wallet-overview-shimmer.component.tsx:51]: Replace this text `<Text variant='small' text='Activities' color='textLight' bold600 />` with a rectangle.

**Comment by Koushith** [comment]
image
image

File: src/screens/wallet-overview/components/wallet-overview-shimmer.component.tsx (hunk 1 above)
Comment (Koushith) [reply src/screens/wallet-overview/components/wallet-overview-shimmer.component.tsx:51]: done. replaced with rectange.

**Comment by Koushith** [comment]
- [x] Fixed navbar responsiveness"

**Review by koshikraj — APPROVED** [review]

**Pull request merged**